tests/rbd_mirror/test_rbd_mirror_snapshot.py

```diff
@@ -41,10 +41,15 @@ def run(**kw):
         snapshot_schedule_level = config.get("snapshot_schedule_level")
         if not snapshot_schedule_level or snapshot_schedule_level == "cluster":
             mirror1.mirror_snapshot_schedule_add()
+            mirror1.mirror_snapshot_schedule_add(interval="24h")
         elif snapshot_schedule_level == "pool":
             mirror1.mirror_snapshot_schedule_add(poolname=poolname)
+            mirror1.mirror_snapshot_schedule_add(poolname=poolname, interval="24h")
         else:
             mirror1.mirror_snapshot_schedule_add(poolname=poolname, imagename=imagename)
+            mirror1.mirror_snapshot_schedule_add(
+                poolname=poolname, imagename=imagename, interval="24h"
+            )
         mirror1.verify_snapshot_schedule(imagespec)
```

Comment:
You should add the same interval parameter to this method as well, so that we verify if the snapshots are actually created based on the interval added in the schedule. @sunilangadi2 could you please pass interval argument here (verify_snapshot_schedule method already accepts interval as an argument) and verify that snapshots are indeed getting created after the specified interval and add the success logs for the same?

Reply:
added verification step for `1h` interval as well.